dotcom-rendering/src/components/YoutubeAtom/YoutubeAtomOverlay.tsx

```diff
@@ -53,6 +53,13 @@ const overlayStyles = css`
 		width: 100%;
 		height: 100%;
 	}
+
+	/* We scale the play icon on hover and focus to indicate it's playable content. */
+	:focus .play-icon,
+	:hover .play-icon {
+		transform: translate(-50%, -50%) scale(1.15);
+		transition-duration: 300ms;
```

Comment:
Should the `transition-duration` apply even when the element is not hovered, or are we specifically trying to get a different animation time?

Reply:
Yeah good point, this must have been preexisting but its smother to not do this. I've added a transition to the element itself for when we stop hovering / focusing.

Reply:
Okay, if it’s added even off the hover and focussed state, you can remove it from here!